src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/impl/FeatureToggleServiceImpl.java

```diff
@@ -0,0 +1,31 @@
+package uk.gov.hmcts.reform.finrem.caseorchestration.service.impl;
+
+import lombok.Getter;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.Features;
+import uk.gov.hmcts.reform.finrem.caseorchestration.service.FeatureToggleService;
+
+import javax.validation.constraints.NotNull;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+
+@Service
+@ConfigurationProperties(prefix = "feature-toggle")
+@Configuration
+@Getter
+public class FeatureToggleServiceImpl implements FeatureToggleService {
```

Comment:
might be a standard on this project already but I would name these classes: 
```suggestion
public class FeatureToggleService implements IFeatureToggleService {
```

Reply:
@shaunganley  Ended up getting rid of the interface entirely and just having a standalone service as it's unlikely we'll be requiring different implementations of the FeatureToggle service